src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+This doc will explain how to use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) to configure your topology mapping. 
+
+This tutorial will give example NerdGraph calls that: 
+* Set up your topology by creating vertices and edges
+* Delete vertices and edges
+* Retrieve topology data
+
+Before using NerdGraph to configure your topology, you should have a basic understanding of: 
+
+* [What Applied Intelligence decisions are](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) and [how topology correlation works](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#topology).
+* The topology structure you're trying to implement. One way to understand this is to use our [service map feature](/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/) to see how entities in your infrastructure relate to each other.  
+* [What NerdGraph is](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) and [how to use the NerdGraph API explorer](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#explorer) to run queries.
+
+## Mutation examples [#mutations]
+
+In NerdGraph, mutations are requests that perform an action ([learn more about NerdGraph terminology](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)). 
+
+In this section, we’ll show you how to use the `aiTopologyCollector` query to create, edit, or delete your topology. In this example, we’ll set up the following service map:
+
+![Topology example](./images/topology-example.png "topology example")
+
+### Create vertices [#create-vertices]
+
+The following mutation creates one or more vertices that represent the entity and the source from which your incident comes from. 
+
+The NerdGraph call below uses the following fields:
+* `accountId`: Your New Relic account ID.
+* `name`: The name of the vertex. This value is case sensitive and must be unique within the graph.
+* `vertexClass`: The classification of this vertex. The class can be `application`, `host`, `cloud service`, `cluster`, or `datastore`. This classification allows your decision logic to restrict your topology-based correlation to restrict vertices matching these classifications.
+* `definingAttributes`: A unique set of attributes (key/value pairs) that match those from incident events. If an incident contains any of the key/value pairs of a vertex’s `definingAttributes`, it’s matched to that vertex. To learn more about attributes and how they’re added, see [Add attributes](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#add-attributes).
```

Comment:
Would it be fair to say definingAttributes are "A unique set of attributes (key/value pairs) that match an incident event's attributes"? 

Or show about "A unique set of key/value pairs that match an incident event's attributes"

I'm not allergic to using the same word more than once in a sentence if it makes the sentence cleaner.